packages/@aws-cdk/aws-events-targets/lib/api-destination.ts

```diff
@@ -0,0 +1,66 @@
+import * as events from '@aws-cdk/aws-events';
+
+/**
+ * Customize the EventBridge Api Destinations Target
+ * @experimental
+ */
+export interface ApiDestinationProps {
+  /**
+   * The event to send
+   *
+   * @default the entire EventBridge event
+   */
+  readonly event?: events.RuleTargetInput;
+
+  /**
+   * The headers that need to be sent as part of request invoking the API Gateway REST API
+   * or EventBridge Api destinations.
+   *
+   * @default none
+   */
+  readonly headerParameters?: { [key: string]: string };
```

Comment:
Why are these here? Aren't they API gateway specific?

Doesn't it make more sense to separate this out into API Gateway destinations and "other" destinations?

Reply:
HttpParameters (headers/body/querystrings) on EventBridge targets are re-used for both ApiGateway and ApiDestinations targets